src/main/java/uk/gov/ons/ssdc/responseoperations/endpoint/PrintSuppliersEndpoint.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.ons.ssdc.responseoperations.endpoint;
+
+import static uk.gov.ons.ssdc.responseoperations.model.entity.UserGroupAuthorisedActivityType.CREATE_PRINT_TEMPLATE;
+
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.ons.ssdc.responseoperations.config.PrintSupplierConfig;
+import uk.gov.ons.ssdc.responseoperations.security.UserIdentity;
+
+@RestController
+@RequestMapping(value = "/api/printsuppliers")
+public class PrintSuppliersEndpoint {
+
+  private final UserIdentity userIdentity;
+  private final PrintSupplierConfig printSupplierConfig;
+
+  public PrintSuppliersEndpoint(
+      UserIdentity userIdentity, PrintSupplierConfig printSupplierConfig) {
+    this.userIdentity = userIdentity;
+    this.printSupplierConfig = printSupplierConfig;
+  }
+
+  @GetMapping
+  public Set<String> getPrintSuppliers(
+      @Value("#{request.getAttribute('userEmail')}") String userEmail) {
+    userIdentity.checkGlobalUserPermission(userEmail, CREATE_PRINT_TEMPLATE);
```

Comment:
maybe your way works, but it seems a bit surprising that this API requires CREATE_PRINT_TEMPLATE permission just to list suppliers. Happy to approve as it is though

Reply:
we are probably going to future-proof ourselves by permssioning it separately, because I think there are future features which also require choosing a print supplier?

Reply:
if there's no other screen that requires the print suppliers, it's OK as is I guess

Reply:
It does make more sense having the `LIST_PRINT_SUPPLIERS` - will update 👍